docs/deploy/is-customize.md

```diff
@@ -0,0 +1,107 @@
+# Customizing an Identity Service deployment
+
+## Increasing resilience
+There are a number of options for increasing the availability and resilience of a deployment.
+
+### Replicas
+During deployment you can specify multiple replicas to increase the resilience of an Identity Service deployment.
+
+To enable multiple replicas, use the following command during the Helm chart deployment:
+
+```bash
+--set alfresco-infrastructure.alfresco-identity-service.keycloak.keycloak.replicas=3
+```
+
+### Clustering
+
+Clustering is another method that can be used to increase the resilience of a deployment. Information on how to configure clustering is available in the following locations:
+
+
+* [High availability and clustering](https://github.com/helm/charts/tree/master/stable/keycloak#high-availability-and-clustering)
+
+
+* [Standalone clustered mode](https://www.keycloak.org/docs/4.5/server_installation/#standalone-clustered-configuration)
+
+
+* [Clustering](https://www.keycloak.org/docs/4.5/server_installation/#_clustering)
+
+
+**Note:** Keycloak recommends that [sticky sessions](https://www.keycloak.org/docs/4.5/server_installation/#sticky-sessions) are used, so be aware if you are using an ingress other than Nginx.
+
+## Client and realm customization
+It is possible to override some parameters set for the default realm and clients during and post-deployment.
+
+### Client redirect URIs
+
+The default redirect URI set during deployment is: `http://localhost*`. Using the `alfresco-identity-service.client.alfresco.redirectUris` property, you can specify a different redirect URI similar to the following example:
+
+```bash
+helm install alfresco-stable/alfresco-infrastructure \
+  --set alfresco-infrastructure.activemq.enabled=false \
+  --set alfresco-infrastructure.nginx-ingress.enabled=true \
+  --set alfresco-infrastructure.alfresco-identity-service.enabled=true \
+  --set alfresco-identity-service.client.alfresco.redirectUris=['\"'http://$DNSNAME*'"\'] \
+  --namespace $DESIREDNAMESPACE
+```
+
+It is also possible to specify multiple redirect URIs using the same property:
+
+```bash
+helm install alfresco-stable/alfresco-infrastructure \
+  --set alfresco-infrastructure.activemq.enabled=false \
+  --set alfresco-infrastructure.nginx-ingress.enabled=true \
+  --set alfresco-infrastructure.alfresco-identity-service.enabled=true \
+  --set alfresco-identity-service.redirectUris=['\"'http://$DNSNAME*'"\'',''\"'http://$DNSNAME1*'"\'',''\"'http://$DNSNAME2*'"\']` \
+  --namespace $DESIREDNAMESPACE
+```
+
+### Customizing the realm
+You can customize a realm during the deployment process or manually in the administration console after you have deployed the Identity Service.
+
+#### Realm customization during deployment
+The below steps advise how to add a realm file to your deployment that contains details of its configuration.
```

Comment:
@m-hulbert I'm probably falling into some great grammar debate but which is preferred? `The below steps` or `The steps below`.  And for my edification why?

Reply:
@jottley Good catch! Grammatically speaking the `the steps below` is likely "more" correct I think, although I've updated it to `the following steps` as we try to avoid above/below. As to the why - it's because it is considered a shortening of `the steps which are shown below` (I think).